STATUS: INLINE SUCCESSFUL
MODIFIED: com.github.kevinsawicki.http.HttpRequestTest.putWithEscapedVarargsQueryParams
MODIFIED FILE: /home/anonymous/projects/tools/reseter-finder/exp/_downloads/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
CLEANER: com.github.kevinsawicki.http.TestGroup100Case0.test224
CLEANER FILE: /home/anonymous/projects/tools/reseter-finder/exp/_downloads/http-request/lib/src/test/java/com/github/kevinsawicki/http/TestGroup100Case0.java
POLLUTER: com.github.kevinsawicki.http.HttpRequestTest.customConnectionFactory
POLLUTER FILE: /home/anonymous/projects/tools/reseter-finder/exp/_downloads/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 2
ELAPSED TIME: 95.65499997138977
==========================
@@ -2899,0 +2899,4 @@
+ {
+     com.github.kevinsawicki.http.HttpRequest.ConnectionFactory connectionFactory0 = null;
+     com.github.kevinsawicki.http.HttpRequest.setConnectionFactory(connectionFactory0);
+ }
